CODEOWNERS

```diff
@@ -46,8 +46,16 @@
 
 /core/services/webhook @samsondav
 
-# Chainlink Functions related services
+# Chainlink Functions
+core/services/functions @bolekk @justinkaseman @KuphJr @pinebit
```

Comment:
These all should have a team name, like for VRF directories.

Reply:
I can't find a team tag for us (I see vrf-team, but not for Functions).  I recall Brad saying the was going to set up a team tag for us, but IDK if that actually happened.

Reply:
We don't have a team tag @bolekk

Reply:
We can amend this later once we do

Reply:
Let's get one then!

Reply:
We can get a proper team before I merge this PR.  No urgency to get this in.